deploy/crds/app.kiegroup.org_v1alpha1_kogitodataindex_cr.yaml

```diff
@@ -21,7 +21,7 @@ spec:
   # details about the connected infinispan
   infinispan:
     # let's leave this burden to KogitoInfra CR to deploy a new Infinispan instance for us
-    useKogitoInfra: true
+    useKogitoInfra: false
```

Comment:
What is the reason for this change?

Reply:
To run the integration tests without the need of having Strimzi and Infinispan in the namespace. The scorecard smoke tests doesn't rely on OLM, thus won't install Strimzi or Infinispan for us, failing the use case.

Reply:
ok, thanks.